packagelist/debian_12_x11.txt

```diff
@@ -75,11 +75,22 @@ build-essential
 command-not-found
 gdb
 git
+libcairo2-dev
+libgif-dev
 libiio-dev
+libjpeg-dev
+libpng-dev
```

Comment:
I'd like to wait until we have user demand for adding any specific graphical libraries just because the size adds up so quickly and they are fairly divergent. Eg, libcairo is not used for QT users, but pulls in some of the gnome gobject stuff.

Reply:
At the very least, we probably don't need the development libraries for them.

Reply:
Several of these showed up because they are dependencies for building `fbv` for the 7100 from source, which in turn populate(s/d) the frame buffer with the boot splash image when Linux switches the video mode as part of taking over the LCD.

What alternatives do we have for rendering into the LCD's frame buffer?

Reply:
1) If we're building `fbv` from source, then these dev packages should be in the build environment, and then the libraries for runtime in the image.
2) Is there any reason to not use `fbi` which is in Debian repos?

Reply:
@ts-kris `fbi` (and `fim`) pull in a lot more packages than `fbv`, including ghostscript/poppler and others that we may not intend to include in the image. `fbi` also needs configuration and parameters. `fim` at least works out of the box, but has the same "tons of dependencies" problem.

Reply:
In that case, see 1) from my previous comment. The end application should be cross-compiled as part of the build system, so those dev libraries should end up in docker and not in the final image.

Reply:
I'd strongly prefer we use something already built by debian/ubuntu if possible so we're not left maintaining the package on newer distributions. For now we should leave out both fbi / fbv if they aren't trivial small installations. 

The TS-7100 is the only platform with a secondary memory for a splash screen, and on the rest of our graphical platforms we still need to solve the lack of splash screen. I'm suspecting this is going to be plymouth which can draw directly to the framebuffer. Once we sort out those usages it may solve this as well.

Reply:
Although these dev packages were in the 7100's Debian image prior to `distro-seed`, they aren't needed as part of this PR.

The next and last 7100 "parity" PR can resolve whether to ship them (preventing a foot-gun if anyone wants to recompile `fbv` from source on their board) or to just use them when building the Debian image. That PR will have a frame-buffer image renderer of some sort for the 7100 so we can get closer to "parity" in minimizing "peek-a-boo" problems during boot.